tests/test_notify_fw_interested_suppliers.py

```diff
@@ -92,16 +94,17 @@ def test_happy_paths(mock_uuid4, run_id, dry_run):
 
     mock_logger = mock.create_autospec(logging.Logger, instance=True)
 
-    assert notify_fw_interested_suppliers(
-        data_api_client=mock_data_api_client,
-        notify_client=mock_notify_client,
-        notify_template_id="8877eeff",
-        framework_slug="g-cloud-99",
-        dry_run=dry_run,
-        stage="production",
-        logger=mock_logger,
-        run_id=run_id,
-    ) == 0
+    with freeze_time('2000-01-04 01:55:55'):
```

Comment:
Given our important framework dates generally fall in BST, shall we use a summertime-y test date to check the localization?

Reply:
Iffff I can find the fixture that controls that, sure...